src/main/java/apoc/graph/document/builder/LabelBuilder.java

```diff
@@ -16,7 +16,7 @@ public LabelBuilder(GraphsConfig config) {
     }
 
     public Label buildLabel(Map<String, Object> obj) {
-        String label = "DocNode"; // Default label
+        String label = "";
```

Comment:
are we making sure that we never end up with `""` as a label, otherwise Neo4j would fail on that.

Reply:
We prepare the data with the `prepareData` method so if the user provide a default Label we'll use it. Otherwise it will fail.

Reply:
The property `defaultLabel` in `GraphConfig` is empty I can use the `DocNode` as default value